### PR TITLE
MGMT-2698 Download service metrics on download_logs

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

**Review by nmagnezi — COMMENTED** [review]
Looks good overall.
Do you have an example `metrics.txt` to show?

File: discovery-infra/download_logs.py [hunk 1]
```diff
@@ -102,6 +103,9 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
     try:
         write_metadata_file(client, cluster, os.path.join(output_folder, 'metdata.json'))
 
+        with suppress(AssertionError, ConnectionError):
```
Comment (nmagnezi) [inline discovery-infra/download_logs.py:106]: Why do we suppress those?

**Review by nmagnezi — COMMENTED** [review]
@YuviGold sent me a `metrics.txt` example. looks good.

One last question regarding this PR (commented)

File: discovery-infra/download_logs.py (hunk 1 above)
Comment (YuviGold) [reply discovery-infra/download_logs.py:106]: Keep running and download everything possible, even if something had failed.

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**